tests/lib/rules/no-sync.js

```diff
@@ -10,14 +10,10 @@ const rule = require("../../../lib/rules/no-sync")
 new RuleTester().run("no-sync", rule, {
     valid: [
         "var foo = fs.foo.foo();",
-        {
-            code: "var foo = fs.fooSync;",
-            options: [{ allowAtRootLevel: true }],
-        },
-        {
-            code: "var foo = fooSync;",
-            options: [{ allowAtRootLevel: true }],
-        },
```

Comment:
why were these cases removed?  its' great to keep them to avoid regressions - they should not be reported (with and without options).

Reply:
They were not removed, just shortened in here:
[`8d8284d` (#364)](https://github.com/eslint-community/eslint-plugin-n/pull/364/commits/8d8284db4ee5aca027dc7056f46efd6ed5ba9257)

I added more test cases here:
[`bb55a55` (#364)](https://github.com/eslint-community/eslint-plugin-n/pull/364/commits/bb55a553ff4acde6d645d837ca17acf468222b84)

Not sure why it looks deleted :thinking: 

---

The only one I did delete was this one, which is because I removed checking anything that is not a `CallExpression` (the first commit).
```js
{
    code: "var foo = fs.fooSync;",
    errors: [
        {
            messageId: "noSync",
            data: { propertyName: "fooSync" },
            type: "MemberExpression",
        },
    ],
},
```

Reply:
I can split the two commits up if thats easier?

Reply:
I reverted it in https://github.com/eslint-community/eslint-plugin-n/pull/386/commits/fd32c39ca5fdc609422d0b81542be43e69c73c8d. you can just cherrypick it. or we can just merge #386 :)

Reply:
Lets merge #386